ChangeLog

```diff
@@ -18,18 +56,75 @@
 	   * `state/cf_users`
 	   * `state/env_data`
 	  (ENT-4863)
-	- The @ character is now allowed in the key of classic arrays defined
-	  by the module protocol (CFE-3099)
-	- Version specific distro classes are now collected by default in Enterprise
-	  (ENT-4752)
-	- Zero bytes in class guards no longer cause crashes (CFE-3028)
+	- Clarified descriptions of io_writtendata and io_readdata (ENT-5127)
+	- Clarified log message about process_count and restart_class being used
+	  concurrently (CFE-208)
+	- Agent runs that hit abortclasses now record results (ENT-2471)
+	- An ID of rhel in os-release file will now define both rhel and redhat
+	  classes (CFE-3140)
+	- Version specific distro classes are now collected by default in
+	  Enterprise (ENT-4752)
+	- redhat_8 and redhat_8_0 are now defined on RHEL 8 (CFE-3140)
+	- Added derived-from-file tag to hard classes based on /etc/redhat-release
+	  (CFE-3140)
+	- Added sys.bootstrap_id policy variable Contains the ID from
+	  `/var/cfengine/bootstrap_id.dat`, if present (CFE-2977)
```

Comment:
s/Contains/contains/ maybe? Not sure why "Contains" would be capitalized.

Reply:
Reworded.